README.template.md

```diff
@@ -386,8 +389,8 @@ files & folders will persist in this Git repository.
 - /var/www/drupal/web/themes/custom
 
 Other changes such as to the `vendor` folder or installed modules are **not**
-persisted, to disk. This is by design as these changes should be managed via
-`composer` and baked into the Drupal Docker image.
+persisted in Git. This is by design as these changes should be managed via
```

Comment:
I'm not sure if you meant for real that changes are not persisted to disk. Based on what little i know of Docker, it's possible that changes are ephemeral and live in the container, not on a volume, so your statement would be accurate. But I don't think that's what your'e going for.

Reply:
I think it's a safe assumption that any manual changes to the vendor/ directory is a bad idea and should be looked at as volatile because running a composer command will wipe out those changes or could be dropped by a build process.